samples/azure-quantum/teleport/README.md

```diff
@@ -0,0 +1,45 @@
+---
+page_type: sample
+author: cgranade
+description: Run quantum teleportation on hardware, using the Azure Quantum service
+ms.author: chgranad@microsoft.com
+ms.date: 01/25/2021
+languages:
+- qsharp
+products:
+- qdk
+- azure-quantum
+---
+
+# Quantum teleportation
```

Comment:
Add "sample" or "example" to H1? See comment above.
This might apply to other H1s as well - I'm reading quickly and not doing an in-depth review.